docs_user/modules/proc_completing-prerequisites-for-migrating-ceph-rgw.adoc

```diff
@@ -1,8 +1,8 @@
 [id="completing-prerequisites-for-migrating-ceph-rgw_{context}"]
 
-= Completing prerequisites for migrating {Ceph} RGW
+= Completing prerequisites for {Ceph} RGW migration
 
-You must complete the following prerequisites before you begin the {Ceph} RGW migration.
+You must complete the following prerequisites before you begin the Ceph Object Gateway (RGW) migration.
```

Comment:
Why did you change {Ceph}?
You could drop "you must" if you like. Prerequisites imply that it's something you must do beforehand anyway.

Reply:
The {Ceph} attribute translates to "Red Hat Ceph Storage". It's not correct to say "Red Hat Ceph Storage Object Gateway"; the term is "Ceph Object Gateway."